src/test/bufferlist.cc

```diff
@@ -1391,6 +1391,24 @@ TEST(BufferList, append_bench) {
   }
 }
 
+TEST(BufferList, append_hole_bench) {
+  constexpr size_t targeted_bl_size = 1048576;
+
+  for (size_t step = 512; step <= 65536; step *= 2) {
+    const utime_t start = ceph_clock_now();
+    constexpr size_t rounds = 80000;
+    for (size_t r = 0; r < rounds; ++r) {
+      ceph::bufferlist bl;
+      while (bl.length() < targeted_bl_size) {
+	bl.append_hole(step);
+      }
+    }
+    cout << rounds << " fills of buffer len " << targeted_bl_size
```

Comment:
this is a performance test with lots of output, and without any EXPECT or ASSERT statements. could you add `GTEST_SKIP()` to disable this test by default?